src/main/java/com/liveramp/captain/request_context/EmptyRequestContext.java

```diff
@@ -8,13 +8,41 @@ public int getPriority() {
     return 0;
   }
 
+  @Override
+  public void setPriority(int priority) {
+    throw new RuntimeException("Cannot set priority on an EmptyRequestContext");
+  }
+
+  @Override public boolean hasPriority() {
```

Comment:
would it be more idiomatic to do `isSetPriority`? or is that just how thrift does it? i don't think it actually matters.